Sources/DataPipeline/Type/Region.swift

```diff
@@ -0,0 +1,18 @@
+import CioInternalCommon
+
+// These URLs are for requesting CDP settings from the CDP API, and must align with server's CDP regional configuration.
+extension Region {
```

Comment:
[Node docs](https://www.customer.io/docs/cdp/sources/connections/servers/node/#eu-data-center) for reference